app/views/signs/show.html.erb

```diff
@@ -48,7 +48,6 @@
     <div class="sign-slides-listing" <%= "id=js-sortable-slides" if can? :update, @sign %>>
       <%= render partial: 'sign_slide', collection: @sign.sign_slides.order(:order).eager_load(:slide) %>
     </div>
-    <%= link_to "+ New Slide", new_slide_path, class: 'btn green add-new-slide', id: 'add-slide-link' if can? :update, @sign %>
```

Comment:
Thanks for the link. I still think we're dealing with different cases here. The article talks about tasks that still strike me as fundamentally related with deployment and are focused on observing/changing the state of a remote app server.

The point of the task here is to change the state of the local dev's environment and has nothing to do with deployment AFAICT. Making changes on a remote server (i.e. cloning a database) is just a side-effect.

So I still think a `rake` task makes more sense and better adheres to Principle of Least Surprise.